.gitignore

```diff
@@ -59,3 +59,8 @@ temp/
 ._*
 .Trashes
 .svn
+
+############
+## Assets
+############
+admin/assets
```

Comment:
Is that new? I don't see any assets added in this PR.

Reply:
Good catch @swissspidy this was from the prior iteration where I had a custom JS handler and REST routes etc, no longer needed so I'll remove.